### PR TITLE
Birthday

### PR DESCRIPTION
**Pull request by dfuentes**
Small worker that runs diamond to collect elb stats.  It logs them as elbs-stats.<metric-name>.

### CODE REVIEW

**Comment by dfuentes** [comment]
pillar values here: https://github.com/Clever/salt/pull/833

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+mkdir -p /etc/diamond
+sed -e "s/<INFLUXDB_USER>/$INFLUXDB_USER/" \
```
Comment (mohit) [inline start.sh:3]: :smiley:

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+image: bradrydzewski/go:1.4
```
Comment (mohit) [inline .drone.yml]: why go?

**Comment by mohit** [comment]
small nit. apart from that looks great!

File: .drone.yml (hunk 2 above)
Comment (dfuentes) [inline .drone.yml]: oops, copy paste.  What's the base image?  We don't actually build anything.

Comment (mohit) [inline .drone.yml]: I think it's called `bradrydzewski/base` but you could just use alpine and comment # it's the smallest image

**Pull request merged**